src/Plugin/rest/resource/ReactParagraphsResource.php

```diff
@@ -129,14 +129,17 @@ public function get($entity_type_id, $bundle) {
       $field_config = $field_config_storage->load("$entity_type_id.$bundle.$field_name");
       if ($field_config && ($plugin = $this->getReactFieldsPlugin($field_config))) {
         $data[$field_name] = $plugin->getFieldInfo($form[$field_name], $field_config);
-        $this->moduleHandler->invokeAll(
+        $altered = $this->moduleHandler->invokeAll(
```

Comment:
if you use `->alter()` method instead, it'll pass the first variable as reference https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21ModuleHandler.php/function/ModuleHandler%3A%3Aalter/8.2.x

Reply:
heres my suggestion: https://github.com/SU-SWS/react_paragraphs/pull/46

Reply:
Works for me. 2:00am doesnt lead to the best decisions.